mibidata/mibi_image.py

```diff
@@ -219,6 +219,18 @@ def __getitem__(self, channels):
         """Equivalent to :meth:`~slice_data`."""
         return self.slice_data(channels)
 
+    def __repr__(self):
+        s = f"{type(self)}\n"
```

Comment:
For consistency please use single-quotes `'`

Reply:
done